From 228ac772366d46eda3a0cc607f01c986fd9f6182 Mon Sep 17 00:00:00 2001
From: gjl <gjl@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Wed, 28 May 2014 08:48:03 +0000
Subject: [PATCH 015/134] 	PR target/61044 	* doc/extend.texi
 (Local Labels): Note that label differences are 	not supported for AVR.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_8-branch@211001 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/ChangeLog       | 6 ++++++
 gcc/doc/extend.texi | 2 ++
 2 files changed, 8 insertions(+)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 0481d58..e38b553 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,9 @@
+2014-05-28  Georg-Johann Lay  <avr@gjlay.de>
+
+	PR target/61044
+	* doc/extend.texi (Local Labels): Note that label differences are
+	not supported for AVR.
+
 2014-05-26  Michael Tautschnig  <mt@debian.org>
 
 	PR target/61249
diff --git a/gcc/doc/extend.texi b/gcc/doc/extend.texi
index c9452c6..29d66a1 100644
--- a/gcc/doc/extend.texi
+++ b/gcc/doc/extend.texi
@@ -375,6 +375,8 @@ goto *(&&foo + array[i]);
 This is more friendly to code living in shared libraries, as it reduces
 the number of dynamic relocations that are needed, and by consequence,
 allows the data to be read-only.
+This alternative with label differences is not supported for the AVR target,
+please use the first approach for AVR programs.
 
 The @code{&&foo} expressions for the same label might have different
 values if the containing function is inlined or cloned.  If a program
-- 
1.8.5.5

